File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/zig/valid.zig
// Fixture — café 日本語 before declarations
const std = @import("std");

/// A point.
const Point = struct {
    x: i32,
    /// Make a point.
    pub fn init(x: i32) Point {
        return Point{ .x = x };
    }
};

const Color = enum { red, blue };

const limit: u32 = 10;

/// Add two numbers.
pub fn add(a: i32, b: i32) i32 {
    return a + b;
}

fn helper(x: i32) i32 {
    return x + 1;
}

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/rust/valid.rs
//! Crate fixture — café 日本語 before declarations
use std::fmt;

/// A point.
pub struct Point {
    x: i32,
}

impl Point {
    /// Make a point.
    pub fn new(x: i32) -> Self { Point { x } }
    fn secret(&self) -> i32 { self.x }
}

pub trait Shape {
    fn area(&self) -> f64;
}

pub enum Color { Red, Blue }

type Id = u32;
pub const LIMIT: u32 = 10;
static COUNTER: u32 = 0;

mod inner {
    pub fn nested() {}
}

fn private_helper() {}
